vajram/extensions/sql/README.md

```
@@ -0,0 +1,164 @@
+    
+    @Provides
+    @Singleton
+    public RowMapper provideRowMapper() {
+        return new DefaultRowMapper();
+    }
+}
+```
+
+### 3. Define Your Entity
+
+**Option 1: Using a Record (Java 16+)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)